app/test/big_blob_test.go

```diff
@@ -83,7 +83,8 @@ func (s *BigBlobSuite) TestErrBlobsTooLarge() {
 			res, err := txClient.SubmitPayForBlob(subCtx, []*blob.Blob{tc.blob}, user.SetGasLimitAndGasPrice(1e9, appconsts.DefaultMinGasPrice))
 			require.Error(t, err)
 			require.NotNil(t, res)
-			require.Equal(t, tc.want, res.Code, res.Logs)
+			// FIXME: assert RawLog once TxStatus supports it.
```

Comment:
I thought the other PR renamed RawLog to error

Reply:
This pr has another backport as a follow-up which is the final state of this work. https://github.com/celestiaorg/celestia-app/pull/3789

Reply:
where i renamed RawLog to error

Reply:
kk so this FIXME is temporary and should be removed via https://github.com/celestiaorg/celestia-app/pull/3789?

Reply:
yes! I'll go through the comments once again to make sure everything is addressed in the follow-up https://github.com/celestiaorg/celestia-app/pull/3789